mittab/libs/structs.py

```diff
@@ -0,0 +1,56 @@
+class DebaterScores(object):
+    def __init__(self, s, t):
+        # (-tot_speaks_deb(debater),
+        #  tot_ranks_deb(debater),
+        #  -single_adjusted_speaks_deb(debater),
+        #  single_adjusted_ranks_deb(debater),
+        #  -double_adjusted_speaks_deb(debater),
+        #  double_adjusted_ranks_deb(debater))
+        self.speaker = s
```

Comment:
Really love this approach, I'm thinking of doing a larger tab logic refactor FYI so I may include this there (and try to do some git-fu to make sure you're credited for the contribution)

Reply:
Thanks@